packages/automerge-repo/src/synchronizer/CollectionSynchronizer.ts

```diff
@@ -35,8 +35,21 @@ export class CollectionSynchronizer extends Synchronizer {
 
   /** Creates a new docSynchronizer and sets it up to propagate messages */
   #initDocSynchronizer(handle: DocHandle<unknown>): DocSynchronizer {
-    const docSynchronizer = new DocSynchronizer(handle)
+    const docSynchronizer = new DocSynchronizer({
+      handle,
+      onLoadSyncState: peerId => {
```

Comment:
This is a little janky; I'll see if I can find a cleaner way to do this but I don't hate it.